docs/src/pages/demos/radio-buttons/radio-buttons.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Radio React component
```

Comment:
Radio buttons

Reply:
I thought the same, but was being cautious since the component is called "Radio".